gui/build.py

```diff
@@ -80,14 +80,21 @@ def do_bundle(options):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
     # create a intaller
+    if IS_WIN:
```

Comment:
I see. :)

Sorry, I have some classes this afternoon. I will send you email after that. Let's fix this issue first. :D